pkg/apps/utils.jsx

```diff
@@ -56,7 +56,7 @@ export const ProgressBar = ({ size, title, data, ariaLabelledBy }) => {
                 {_("Waiting for other programs to finish using the package manager...")}
             </SplitItem>
             <SplitItem>
-                <Spinner isSVG size="md" />
+                <Spinner size="md" />
```

Comment:
"Remove isSvg property from Spinner component", is that already true for PF4? Could this be split out?

Reply:
The default for PF4 is false. https://www.patternfly.org/v4/components/spinner